src/App.css

```diff
@@ -3,11 +3,11 @@
   padding: 0;
   box-sizing: border-box;
 }
-
+/* Resetear los paddings que los navegadores tiene por defecto y el border box...() */
 html {
   font-size: 62.5%;
 }
-
+/* Ajustar las medidas relativas como el rem (root element) 62.5% = 10  y de manera uniforme en el proyecto*/
```

Comment:
It is really good you added a comment here 

No need to change it but remember to always use English when coding, including comments